history/cultures/ancient_lore.txt

```diff
@@ -5,6 +5,8 @@
 	discover_innovation = innovation_plenary_assemblies 
 	discover_innovation = innovation_casus_belli 
 	#
+	discover_innovation = innovation_giants
```

Comment:
IIRC you said that Giants units are only available to culture that has much bigger unit than their base unit. In that case, ancient could be removed from that category.